### DISCUSSION TITLE
Multiclassification example for n_classes > 2

### DISCUSSION THREAD

**Discussion by charlesity** [cornellius-gp/gpytorch#2438] [Q&A]
Hi everyone,

I am pretty new to gy-pytorch and I am hoping that someone can point me to an example implementation for multiclassification setting for more than 2 classes. 
I tried changing the likelihood function in this https://docs.gpytorch.ai/en/stable/examples/04_Variational_and_Approximate_GPs/Non_Gaussian_Likelihoods.html but i kept getting to softmax without any success

- Charles

- C
